CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+## 2021-04-12 unftp-auth-jsonfile renamed to unftp-auth-json
```

Comment:
Your name is better. However, we can’t ever remove the crate on crates.io so its gonna sit there. I’d prefer if we just stick with the old name.

Reply:
We can release a 0.2.0 version of `unftp-auth-jsonfile`, that has only a `README.md` to use `unftp-auth-json`.
On that note if we would do that, would this be a new breaking version of `libunftp` or not?

Reply:
personally I think jsonfile was a fine name, as it makes clear that the credentials are read from a json formatted file, as 'json' by itself is a format used in many places

Reply:
Naming is changed back to the original